app/models/schedule.rb

```diff
@@ -137,26 +137,43 @@ def clean_up_task_jobs
     end
   end
 
-  def timeline
-    self.root_task.self_and_descendants.collect do |task|
-      
-      unless task.sprout.nil?
-        rhizome_type = task.sprout.type.titlecase
-        number = task.sprout.rhizome_eid
-      end  
-
-      # this does not have the ramp up time for the thermostat included
-      [
-        "#{task.id}", 
-        "#{task.type_name_display} #{rhizome_type} #{number}", 
-        "#{rhizome_type}",
-        task.trigger_start*1000,
-        nil,
-        task.duration*1000,
-        nil,
-        ""  
-      ]
-    end 
+  # Provides a datatable representing the Schedule for use with a Google Chart Gantt chart
+  # @return [GoogleVisualr::DataTable] The datatable
+  def to_gantt_data
+    timeline = []
+    unless self.root_task.nil?
+      timeline = self.root_task.self_and_descendants.collect do |task|
+        task.to_gantt_data
+      end
+      timeline.flatten!(1)
+    end
+
+    chart_data = GoogleVisualr::DataTable.new
+    chart_data.new_column('string', 'Target ID')
+    chart_data.new_column('string', 'Task Name')
+    chart_data.new_column('string', 'Resource')
+    chart_data.new_column('date', 'Start Date')
+    chart_data.new_column('date', 'End Date')
+    chart_data.new_column('number', 'Duration')
+    chart_data.new_column('number', 'Percent Complete')
+    chart_data.new_column('string', 'Dependencies')
+    chart_data.add_rows(timeline)
+    chart_data
+  end
```

Comment:
Unfortunately, it doesn't look like the Gantt chart is respecting the same sorts of options you can pass into many of the other charts (bar, etc.). It seems to ignore any axis formatting I pass in. I've verified that it's the charts rather than just the gem by messing around with the provided chart examples in jsFiddle.

Kinda sucks, but maybe we can spend more time later messing with stuff directly through Javascript. I'd like to remove the "Percentage complete" part in addition to getting the axes looking better.

In the meantime, I've added a bit of text explaining what the times mean. Hours *will* show up depending on the length of the overall schedule and the current time, but it's largely left up to Google Charts as to whether that should happen.